Documentation/commands.md

```diff
@@ -132,6 +132,23 @@ rkt: fetching image from https://github.com/coreos/etcd/releases/download/v2.0.0
 sha512-fa1cb92dc276b0f9bedf87981e61ecde
 ```
 
+#### Fetch from a Docker registry
+
+If you want to run an existing Docker image, you can fetch from a Docker registry. Rocket will download and convert the image to ACI.
+
+```
+$ rkt -insecure-skip-verify fetch docker://busybox
+rkt: fetching image from docker://busybox
+rkt: warning: signature verification has been disabled
+Downloading layer: 4986bf8c15363d1c5d15512d5266f8777bfba4974ac56e3270e7760f6f0a8125
+Downloading layer: ea13149945cb6b1e746bf28032f02e9b5a793523481a0a18645fc77ad53c4ea2
+Downloading layer: df7546f9f060a2268024c8a230d8639878585defcc1bc6f79d2728a13957871b
+Downloading layer: 511136ea3c5a64f264b78b5433614aec563103b4d4702f3ba7d4d2698e22c158
+sha512-c4010045aec65aefa74770ef2bb648d9
+```
+
+Signature verification for Docker images is not supported.
```

Comment:
We don't support it because it's simply not supported by Docker images, right?
I changed the wording to make it more clear.